Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/EditPage.java

```diff
@@ -35,6 +37,15 @@ void addRow(List<WebElement> tableRows, String title, WebElement dialog) {
         for (WebElement tableRow : tableRows) {
             if (Browser.getCellDataByRow(tableRow, 0).equals(title)) {
                 clickLinkOfTableRow(tableRow);
+
+                // Hold up a second…
+                try {
+                    Thread.sleep(1*1000);
+                } catch (InterruptedException e) {
+                    Logger logger = LogManager.getLogger(Browser.class);
+                    logger.error(e.getMessage());
```

Comment:
Stacktrace would be also useful:
logger.error(e.getMessage()), e;

Reply:
What for? That's a super rare special case that this exception gets thrown, right?

Reply:
But if it is thrown it would be nice to be able just click in console to get to the place where it was thrown :)

Reply:
Are you telling me you run the Selenium tests locally? :P

Reply:
Of course :) Quiet often to find out why certain specific selenium test fails. I do it for me and for other developers :D